composer.json

```diff
@@ -17,28 +16,53 @@
   ],
   "scripts": {
     "dev": "php -S 0.0.0.0:8000",
-    "analyse": "vendor/bin/phpstan analyse --ansi",
+    "phpstan": "vendor/bin/phpstan analyse --ansi",
     "rector": "vendor/bin/rector process --dry-run --ansi",
     "rector:fix": "vendor/bin/rector process --ansi",
     "style": "vendor/bin/ecs check --ansi",
-    "style:fix": "vendor/bin/ecs check --fix --ansi"
+    "style:fix": "vendor/bin/ecs check --fix --ansi",
+    "pest": "./vendor/bin/pest --stop-on-defect --colors=always",
+    "pest:parallel": "./vendor/bin/pest --colors=always --parallel",
+    "pest:profiling": "./vendor/bin/pest --colors=always --profile",
+    "pest:coverage": "./vendor/bin/pest --colors=always --coverage",
+    "pest:type-coverage": "./vendor/bin/pest --colors=always --type-coverage --min=99",
+    "test": [
+      "@test:style",
+      "@test:rector",
+      "@phpstan",
+      "@pest:type-coverage",
+      "@pest:parallel"
+    ],
+    "core:update": [
+      "composer update --no-interaction --prefer-stable",
+      "composer bump --ansi",
+      "composer validate --strict --ansi",
+      "composer audit --ansi"
+    ]
   },
   "require": {
-    "php": ">=8.1",
+    "php": ">=8.2",
     "composer-runtime-api": "^2.2",
-    "adhocore/cli": "^v1.7.1",
-    "nikic/php-parser": "^v4.0"
+    "adhocore/cli": "^1.7.2",
+    "nikic/php-parser": "^5.2"
```

Comment:
Using `php-parser` v5 is problematic because [`rector` uses `php-parser` v4](https://github.com/rectorphp/rector-src/blob/c5940d2cc1adbe39945e0a0f9029823a883671e6/composer.json#L23).
Projects having both `php-icons` and `rector` would have conflicting `php-parser` versions unfortunately.

That's why I had set it to v4 for now.

Reply:
Okay I'm gonna rollback too, it's related to PEST :)